shell/server/accounts/ldap/ldap-server.js

```diff
@@ -23,7 +23,7 @@ LDAP_DEFAULTS = {
  @class LDAP
  @constructor
  */
-let LDAP = function () {
+LDAP = function () {
```

Comment:
Please find a way to design this code such that we don't make db.js depend on this global.